src/CrosschainDeployScript.sol

```diff
@@ -10,9 +10,11 @@ import {ICrosschainDeployAdapter} from "./interfaces/CrosschainDeployAdapterInte
  * @author ChainSafe Systems
  */
 contract CrosschainDeployScript is Script {
```

Comment:
maybe it makes sense to have "reset" method to reset state (if you want to deploy same contract with different params)